.github/workflows/deploy.yml

```diff
@@ -285,14 +285,14 @@ jobs:
         # v4
         uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02
         with:
-          name: ${{ fromJSON(env.SHOULD_DEPLOY_PRODUCTION) && 'desktop-sourcemaps-artifact' || 'desktop-staging-sourcemaps-artifact' }}
+          name: desktop-sourcemaps-artifact
```

Comment:
I think these artifact names will not collide anymore, since we are no longer re-using releases for staging and prod